### PR TITLE
Splits terminal buttons

### PR DESCRIPTION
**Pull request by pastuxso**
Splits terminal buttons to have separate business logic and conditions.

### CODE REVIEW

File: src/client/components/terminal/share.ts [hunk 1]
```diff
@@ -8,13 +8,22 @@ import { SaveIcon } from '../icons/save'
 @customElement('share-cell')
```
Comment (sourishkrout) [inline src/client/components/terminal/share.ts:8]: Is this still an appropriate name for the component?

**Review by sourishkrout — APPROVED** [review]
✅ LGTM. Just one nit, perhaps renaming the component?

File: src/client/components/terminal/share.ts (hunk 1 above)
Comment (pastuxso) [reply src/client/components/terminal/share.ts:8]: Sure, I think it should be more generic. Let me rename it.

Comment (sourishkrout) [reply src/client/components/terminal/share.ts:8]: Something like `ActionButton` might be a good choice

**Pull request merged**